src/directives/gmf-map.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @fileoverview This file provides the "map" directive
+ * for GeoMapFish applications.
+ *
+ * Example:
+ *
+ * <ngeo-gmf-map ngeo-gmf-map-map="::mainCtrl.map"></ngeo-gmf-map>
+ */
+goog.provide('ngeo.gmf.mapDirective');
+
+goog.require('goog.asserts');
+goog.require('ngeo.Debounce');
+goog.require('ngeo.Location');
+goog.require('ngeo.mapDirective');
+
+
+/**
+ * @return {angular.Directive} The Directive Definition Object.
+ * @ngInject
+ */
+ngeo.gmf.mapDirective = function() {
+  return {
+    scope: {
+      'map': '=ngeoGmfMapMap'
+    },
+    bindToController: true,
```

Comment:
> Do you have scope: {}?

Yes, bindToController works on new scope. You either need `scope: true` or `scope: {}` to make it work.
In your case `scope: {}`.
Note that you can have both :

    scope: {toto:'='},
    bindToController: {tata: '='}